File: fsharp-giraffe-server/OpenAPI/src/model/RateLimitedError.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.AnyType
open OpenAPI.Model.CodedError

module RateLimitedError = 

  //#region RateLimitedError


  type RateLimitedError = {
    Result : AnyType;
    Msg : AnyType;
    Code : AnyType;
  }
  //#endregion
  

File: scala-finch/src/main/scala/org/openapitools/models/RateLimitedError.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import org.openapitools.models.AnyType
import org.openapitools.models.CodedError

/**
 * 
 * @param result 
 * @param msg 
 * @param code 
 */
case class RateLimitedError(result: AnyType,
                msg: AnyType,
                code: Option[AnyType]
                )

object RateLimitedError {
    /**
     * Creates the codec for converting RateLimitedError from and to JSON.
     */
    implicit val decoder: Decoder[RateLimitedError] = deriveDecoder
    implicit val encoder: ObjectEncoder[RateLimitedError] = deriveEncoder
}
